libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -185,6 +185,7 @@
         NOTIFICATION_UNAPPROVED,
         NOTIFICATION_LIKED,
         NOTIFICATION_QUICK_ACTIONS_LIKED,
+        NOTIFICATION_QUICK_ACTIONS_QUICKACTION_TOUCH,
```

Comment:
Should we use `NOTIFICATION_QUICK_ACTIONS_QUICKACTION_TOUCHED`?

Reply:
agreed - 64a386c062